src/main/resources/application.properties

```diff
@@ -38,6 +38,7 @@ application.usageControlVersion=platoon
 application.ecc.protocol=https
 application.ecc.host=localhost
 application.ecc.port=8887
+application.ecc.wss-port=8098
```

Comment:
For which this wss port in ECC should be used?
Do we have this one documented in readme of the DataApp in WSS section?
Will this one have impact on dockerized environment?

Reply:
This property already exists in ECC, and it is used for WSS flow, and now it must be present in DataApp. I added it in README for time being, but the whole file should be modified, due to new things that were done in the previous period.
It will have an impact on the dockerized environment since we need this property in order to establish WSS flow, so it needs to be added also there.